codegen/templates/endpoint.tmpl

```diff
@@ -1,6 +1,20 @@
 {{/* template to render gateway http endpoint code */ -}}
-
-package {{.PackageName}}
+{{- $instance := .Instance }}
+package {{$instance.PackageInfo.PackageName}}
+
+{{- $workflow := .WorkflowName }}
+{{- $reqHeaderMap := .ReqHeaderMap }}
+{{- $reqHeaderMapKeys := .ReqHeaderMapKeys }}
+{{- $resHeaderMap := .ResHeaderMap }}
+{{- $resHeaderMapKeys := .ResHeaderMapKeys }}
+{{- $clientName := title .ClientName }}
+{{- $serviceMethod := printf "%s%s" .Method.ThriftService .Method.Name }}
```

Comment:
Yeah I picked the endpoint (not tchannel endpoint) convention just because I don't like computing stuff in the templates. I've updated it per your suggestion.